dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/AbstractClouderaManagerTask.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.cloudera.manager;
+
+import com.google.common.io.ByteSink;
+import com.google.edwmigration.dumper.application.dumper.handle.Handle;
+import com.google.edwmigration.dumper.application.dumper.task.AbstractTask;
+import com.google.edwmigration.dumper.application.dumper.task.TaskRunContext;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+abstract class AbstractClouderaManagerTask extends AbstractTask<Void> {
+  public AbstractClouderaManagerTask(String targetPath) {
+    super(targetPath);
+  }
+
+  @CheckForNull
+  @Override
+  protected final Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
+      throws Exception {
+    return doRun(context, sink, (ClouderaManagerHandle) handle);
+  }
+
+  protected abstract Void doRun(
+      TaskRunContext context, @Nonnull ByteSink sink, @Nonnull ClouderaManagerHandle handle)
+      throws Exception;
+
+  protected final String jsonToJsonl(String json) {
+    json = json.replaceAll("\n", "");
+    json = json.replaceAll("\r", "");
+    return json;
+  }
```

Comment:
This, and its usages, feel a little bit fragile. Do we think it's worth doing an unchecked Jackson parse into a tree, and then a re-stringify with the no-pretty-print flag set?

Reply:
I understand that this code might looks weird when you see it first time. However it works according to specification https://jsonlines.org/.

Using Jackson for this purpose might be more costly in terms of footprint, but as I understand the Dumper use case the additional footprint of Jackson usage will not be visible in real usage.  So, I agree, let's use Jackson to keep the code more natural for developers. Also we have unit-test for `jsonl` so it's quite safe change.

Reply:
I think we can do parsing of string into `JsonNode` first and then write it back as `json`. Is it what you mean?

Reply:
done